httpbinding/query.go

```diff
@@ -14,7 +14,7 @@ type QueryValue struct {
 	append bool
 }
 
-func newQueryValue(query url.Values, key string, append bool) QueryValue {
+func NewQueryValue(query url.Values, key string, append bool) QueryValue {
```

Comment:
Did you intend to export this function? It doesn't look like it is used by any other package.

Reply:
Yes I intended to - I'll be using it in the query serializer